create_xlinkdocs.sh

```diff
@@ -2,111 +2,81 @@
 # Build cross linked doxygen documents and load into buildbot public_html website
 
 SCRIPT_DIR=$(cd "$(dirname "$0")"; pwd)
-source ${SCRIPT_DIR}/settings.cfg.sh
-source ${LSSTSW}/bin/setup.sh
+source "${SCRIPT_DIR}/settings.cfg.sh"
+source "${LSSTSW}/bin/setup.sh"
 
 usage() {
-    echo "Usage: $0 --type <type> --user <remote user on doxy host> --host <remote doxy host> --path  <remote doxy docs path>"
+    echo "Usage: $0 --type <type> --host <remote doxy host> --path <doxy docs path>"
     echo "Build crosslinked doxygen documentation and install on LSST website."
     echo "             type: either <git-branch>,  \"stable\", or \"beta\""
-    echo "             user: remote user account with access to directory containing the DM doxygen documentation"
-    echo "             host: remote system hosting the publicly accessible DM doxygen documentation"
-    echo "             path: actual remote path to the publicly accessible DM doxygen documentation"
-    echo "Example: $0 --type master --user buildbot --host lsst-dev.ncsa.illinois.edu --path /lsst/home/buildbot/public_html/doxygen"
-    echo "Example: $0 --type Winter2012 --user buildbot --host lsst-dev.ncsa.illinois.edu --path /lsst/home/buildbot/public_html/doxygen"
-    echo "Example: $0 --type stable --user buildbot --host lsst-dev.ncsa.illinois.edu --path /lsst/home/buildbot/public_html/doxygen"
-    exit $BUILDBOT_FAILURE
+    echo "             path: actual path to the publicly accessible DM doxygen documentation"
+    echo "Example: $0 --type master --path /home/foo/public_html/doxygen"
+    echo "Example: $0 --type Winter2012 ---path /home/foo/public_html/doxygen"
+    echo "Example: $0 --type stable --path /home/foo/public_html/doxygen"
+    exit "$BUILDBOT_FAILURE"
 }
 
-#-----------------------------------------------------------------------------
-#                         W A R N I N G
-# The following test process will install the documentation of latest master build
-# into ~buildbot/public_html/doxygen .
-#                         W A R N I N G
-#  To manually invoke this buildbot script,  you need to have your ssh public
-#  key installed in ~buildbot/.ssh/authorized_keys  and then do:
-# % <setup eups>
-# % cd <your work dir>
-# % ~buildbot/RHEL6/scripts/create_xlinkdocs.sh --type master --user buildbot --host lsst-dev.ncsa.illinois.edu --path /lsst/home/buildbot/public_html/doxygen
-#-----------------------------------------------------------------------------
 echo "LSSTSW_BUILD_DIR: $LSSTSW_BUILD_DIR"
 
-options=(getopt --long type:,user:,host:,directory: -- "$@")
+# shellcheck disable=SC2034 disable=SC2054
+options=(getopt --long type:,user:,host:,directory:,ssh -- "$@")
```

Comment:
This seems to still have "host" in it, plus a "ssh" flag that is otherwise ignored.  Also, "directory" is used instead of "path".